schemas/skills/samples/complex-skillmanifest-2.1.preview-1.json

```diff
@@ -106,6 +106,18 @@
         "message": {
             "type": "message",
             "description": "Receives the user's' utterance and attempts to resolve it using the skill's LU models"
+        },
+        "typing": {
+            "type": "typing"
+        },
+        "conversationUpdate": {
```

Comment:
Not sure if this truly represents a conversationUpdate activity, to be able to cast this to an OOTB SDK type it should represent https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Schema/IConversationUpdateActivity.cs.
Value is not set for conversationUpdates for example.